src/components/Menu/MenuItem.tsx

```diff
@@ -229,6 +233,7 @@ class MenuItem extends AutoControlledComponent<ReactProps<MenuItemProps>, MenuIt
           {Menu.create(menu, {
             defaultProps: {
               accessibility: submenuBehavior,
+              color,
               vertical: true,
               primary,
```

Comment:
Shouldn't we remove `primary` and `secondary` props

Reply:
we can do that, but I wanted to have a separate PR for it; it also needs to be discussed again; I'll update the comment after we sync